tools/topology/topology2/production/tplg-targets-ace2.cmake

```diff
@@ -91,6 +91,14 @@ DMIC0_ID=3,DMIC1_ID=4,HDMI1_ID=5,HDMI2_ID=6,HDMI3_ID=7,DMIC0_ENHANCED_CAPTURE=tr
 EFX_DMIC0_TDFB_PARAMS=line2_generic_pm10deg,EFX_DMIC0_DRC_PARAMS=dmic_default,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=sof-lnl-cs42l43-l0-cs35l56-l23-2ch.bin"
 
+"cavs-sdw\;sof-lnl-cs42l43-l0-cs35l56-l23-4ch\;PLATFORM=lnl,NUM_SDW_AMP_LINKS=2,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack,NUM_DMICS=4,\
+PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
```

Comment:
Just looking and comparing to the 2 channel version, should I have enabled the PDM on the MICs?

Reply:
The 2 channel DMIC uses PDM0_DMIC and PDM0_MIC_A_ENABLE and PDM0_MIC_B_ENABLE are enabled by default. @singalsu Please correct me if I am wrong.

Reply:
Aaah great thanks!